dynkt-osgi/pom.xml

```diff
@@ -10,18 +10,12 @@
 	</parent>
 
 	<artifactId>dynkt-jsr223-osgi</artifactId>
-	<packaging>jar</packaging>
+	<packaging>bundle</packaging>
 
 	<name>DynKT (Dynamic Kotlin) OSGi Bundle</name>
 	<description>This is a JSR-223 scripting engine for the language "Kotlin".</description>
 	<url>https://github.com/xafero/dynkt</url>
 
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.6</maven.compiler.source>
-		<maven.compiler.target>1.6</maven.compiler.target>		
-	</properties>
-
```

Comment:
These were duplicating the properties from the parent pom. That's not necessary.